### DISCUSSION TITLE
Boot from EMMC

### DISCUSSION THREAD

**Discussion by a-dasler** [devmfc/debian-on-amlogic#4] [Q&A] [answered]
Sorry, I don't know much about Linux. I would like to use emmc boot from my s905x.
Is there a way to change the bootloader under my responsibility?
And, if there is no way, then how can I make ./aml-multiboot-setup.sh run from the root folder?
Sorry for my ignorance.
Debian img

**Comment by devmfc** [comment, marked as answer]
I don't understand what you mean?

Does ./aml-multiboot-setup.sh not work for you? Or do you want to install Debian on eMMC?

In either way, there is no change of the bootloader necessary.

If you want to install Debian to your eMMC you could use something like https://gist.github.com/devmfc/d6b3a5cfbb2802b61bea5b799a03498b 

But use at your own risk. I have not fully tested that script and I never used it on a s905x, but that should not make much of a difference.

**Reply by afgp87** [reply]
Thank you for all your work, I want to install it in the emmc, but get this error. I don't need an android or a second boot 
How I can install in the emmc mainly I want to used as a server for klipper 
![Screenshot_20240926_161103_Mobile SSH.jpg](https://github.com/user-attachments/assets/0cce4a48-4b7b-48b4-81f0-ae77014cac20)

**Comment by a-dasler** [comment]
Yes, I want to install the system on emmc, and boot from emmc. So far I can install the system on emmc, but boot does not work without an SD card. Simply running ./aml-multiboot-setup.sh does not help, nothing is loaded at all except the SD card. You have written that you need to run ./aml-multiboot-setup.sh during boot from /root. I thought this would help.
I’ll definitely try your script this evening and post the results.
Thank you!

**Comment by a-dasler** [comment]
Thank you very much, just what the doctor ordered!
I like your distribution the most!

**Comment by devmfc** [comment]
Happy to hear that.
You're welcome!

Can I ask what tvbox you are using? (So that I can put it on the list of compatible boxes if not already on the list)

**Comment by a-dasler** [comment]
This is some kind of noname, analogue of Beelink Mini MXIII-II, s905x, 2-16
